pkg/asset/mock/gcpclient_generated.go

```diff
@@ -0,0 +1,65 @@
+// Code generated by MockGen. DO NOT EDIT.
```

Comment:
this fine should be in `pkg/asset/installconfig/gcp/mock` right???

Reply:
like https://github.com/openshift/installer/blob/master/pkg/types/openstack/validation/mock/validvaluesfetcher_generated.go

Reply:
I was just putting it in the same dir as https://github.com/openshift/installer/blob/master/pkg/asset/mock/filefetcher_generated.go

I am fine to move but think it is also fine to just have one dir to hold them all. would be a little less messy. Let me know whether to move or not.

What I feel more strongly about is that these mocks should have their own packages. mock_gcpclient, mock_filefetcher, etc in order to avoid collisions. I plan on opening a separate PR for this.

Reply:
> I am fine to move but think it is also fine to just have one dir to hold them all. would be a little less messy. Let me know whether to move or not.

i don't think one directory gets us anything, the locality is better for managing ownership. And i would like to move this to `pkg/asset/installconfig/gcp/mock`

> What I feel more strongly about is that these mocks should have their own packages. mock_gcpclient, mock_filefetcher, etc in order to avoid collisions.

mocks for **one** package can and should in a single package, because there is no chance of collision between interfaces in same package. so the per mock package is overkill.